sample_data/china_by_province_with_chinese_taipei_since_1998/china_population_2021.csv

```diff
@@ -0,0 +1,34 @@
+NAME_PINGY,Cartogram Data (eg. Population),Color,Inset,Label
+Beijing Shi,21893095,,,
+Tianjin Shi,13866009,,,
+Hebei Sheng,74610235,,,
+Shanxi Sheng,34915616,,,
+Neimenggu Zizhiqu,24049155,,,
+Liaoning Sheng,42591407,,,
+Jilin Sheng,24073453,,,
+Heilongjiang Sheng,31850088,,,
+Shanghai Shi,24870895,,,
+Jiangsu Sheng,84748016,,,
+Zhejiang Sheng,64567588,,,
+Anhui Sheng,61027171,,,
+Fujian Sheng,41540086,,,
+Jiangxi Sheng,45188635,,,
+Shandong Sheng,101527453,,,
+Henan Sheng,99365519,,,
+Hubei Sheng,57752557,,,
+Hunan Sheng,66444864,,,
+Guangdong Sheng,126012510,,,
+Guangxizhuangzu Zizhiqu,50126804,,,
+Hainan Sheng,10081232,,,
+Sichuan Sheng,83674866,,,
+Guizhou Sheng,38562148,,,
+Yunnan Sheng,47209277,,,
+Xizang Zizhiqu,3648100,,,
+Shaanxi Sheng,39528999,,,
+Gansu Sheng,25019831,,,
+Qinghai Sheng,5923957,,,
+Ningxiahuizu Zizhiqu,7202654,,,
+Xinjiangweiwu'erzu Zizhiqu,25852345,,,
+Taiwan Sheng,23904454,,,
```

Comment:
The population here seems to come from the reference link's (https://www.worldometers.info/world-population/taiwan-population/) live population. Using live population is not a good choice considering the rest of the regions' population is from 2020. I suggest we use Taiwan's 2020 population which is 23,816,775 (found at the same link bit below).